### ISSUE TITLE
Unable to run topologicpy 0.4.67 on Linux

### ISSUE THREAD

**Issue by felipeacsi** [wassimj/topologicpy#46]
Hello,

First and foremost, thank you for all the work on this software. It's really promising.

I can install topologicpy with `pip install -U topologicpy` on Linux, but when I try to import it I get this error:

```
Python 3.11.6 (main, Nov 14 2023, 09:36:21) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import topologicpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.virtualenvs/dev/lib/python3.11/site-packages/topologicpy/__init__.py", line 37, in <module>
    import topologic
ImportError: libuuid-40d0abac.so.1.3.0: cannot open shared object file: No such file or directory
```

The last working version is 0.4.65. I've also tested in a clean virtual environment and I got the same error.

Do you know what is happening?

Thanks in advance.

Felipe

**Comment by wassimj** [comment]
Please do me a favour: Install the last working version, copy out the bin folder from it. Install the latest version. Copy over the bin folder to overwrite the installed one and see if it works. Thanks

**Comment by wassimj** [comment]
I believe this is solved with the new version 0.4.68. Can you please test and report back here? Thank you!
![Screenshot](https://github.com/wassimj/topologicpy/assets/6302909/f51fc919-d632-40ad-970e-5167dd724bc2)

**Comment by felipeacsi** [comment]
Now it is working with version 0.4.68. Thank you!
